### DISCUSSION TITLE
Connecting from external lighttpd server

### DISCUSSION THREAD

**Discussion by FenrirWolfwood** [Lissy93/dashy#850] [Help and Support]
Hi, maybe not directly a dashy issue, but I'm quite lost.

I have dashy running on a server which I can already access through http://10.0.2.7:4000/ and also have a lighttpd server on the same network from which I want to access dashy.

I don't fully understand lighttpd so I just copied a working config file from a local service and made some adjustments.

```
server.modules += ( "mod_proxy" )

$HTTP["url"] =~ "^/dashy/" {
  proxy.header = (
    "map-host-request" => ( "-" => "10.0.2.7" ),
    "map-urlpath" => ( "/dashy" => "", "/dashy/" => "/" ),
    "https-remap" => "enable"
  )
  proxy.server = ( "" => ( ( "host" => "10.0.2.7", "port" => "4000") ) )
}

$HTTP["url"] =~ "^/dashy$" {
    url.redirect = ( "^/dashy$" => "/dashy/" )
```

When I try to access http://my-server/dashy it seems to connect, but doesn't pass from the "dashy loading..." page and I don't understand why.

Little help? I really don't know what I'm doing wrong.

**Comment by lissy93** [comment]
I'm not super familiar with Lighttpd...

But if you could also share the error that you see in the browser console ([here's how](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#how-to-open-browser-console) if you're unsure), then that might shine a  bit of light on the issue. I guess it's a 404 from not being able to find the required resources, or a 401 if you've got auth.

Edit: actually, running from a sub-path (e.g. `example.com/dashy` instead of `dashy.example.com`) has had a few issues. Though not officially supported, I know some users have gotten it working, but the way Vue compiles made it really hard.

**Comment by FenrirWolfwood** [comment]
Not sure if this is what you are referring to, but I got to the browser inspection tools and found this difference.
| Connecting directly | Connecting through Lighttpd |
|-:|-|
|![imagen](https://user-images.githubusercontent.com/100745429/185001747-aeb833d9-3aa8-4431-8599-622fb3a46f05.png)|![Screenshot_20220817_012320](https://user-images.githubusercontent.com/100745429/185001879-284eb1dc-8e72-4c8c-8bed-c8894c523946.png)|
| dashy:4:1 -->  |![Screenshot_20220817_012540](https://user-images.githubusercontent.com/100745429/185002169-9adbcfb0-769f-4e9f-b957-10825df99e7a.png)|

Looks like lighttpd can redirect wright but after that doesn't know how to refresh correctly page contents? Doesn't find again the path maybe?

PS: Thanks a lot for your help!!
